src/Animated.test.js

```diff
@@ -0,0 +1,106 @@
+import Animated, { Easing } from './Animated';
+import ReanimatedModule from './ReanimatedModule';
+import AnimatedNode from './core/AnimatedNode';
+
+jest.mock('./ReanimatedEventEmitter');
+jest.mock('./ReanimatedModule');
+jest.mock('./derived/evaluateOnce');
+
+const { Value, timing, spring, decay } = Animated;
+describe('Reanimated backward compatible API', () => {
+  beforeEach(() => {
+    let nodesCreated = 0;
+    ReanimatedModule.createNode = () => nodesCreated++;
+    ReanimatedModule.dropNode = () => nodesCreated--;
```

Comment:
this does not reflect the number of created nodes if you decrement it. Maybe rename to `activeNodes` or just `numberOfNodes`